### DISCUSSION TITLE
[求助]用大内存U盘装绝区零怎么样

### DISCUSSION THREAD

**Discussion by shidijun** [share121/inter-knot#4589] [绝区零]
因为电脑各配件都是古董，想以后再配电脑，如果我用大内存U盘装绝区零体验会好吗？

**Comment by shidijun** [comment]
没热度 顶一下

**Comment by EP-office** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/dfc7ccf0203331b18508e05f4ccd7834_8488698876932024893.png" alt="本-疯狂计算" width="75" >不到啊

**Reply by shidijun** [reply]
呜呜呜<img src="https://bbs-static.miyoushe.com/static/2023/11/16/189d73f83e4892766ac8197f2df156d0_3183903856565806928.png" alt="比利-呜呜呜" width="75" >

**Comment by mars070423** [comment]
不行吧？我记得u盘读取数据比较慢，可能会出现你传送要加载两年半到情况

**Reply by shidijun** [reply]
看到一个100MB/s的 所以才有了这种想法 理想状态读完zzz10min都用不了 可我怕的是虚假宣传<img src="https://bbs-static.miyoushe.com/static/2024/05/13/0b9b4077bb63bfd00c6d30e664a16df4_8690624140861603946.png" alt="赛诺-为难" width="75" >

**Reply by Yangchuwen** [reply]
> 看到一个100MB/s的 所以才有了这种想法 理想状态读完zzz10min都用不了 可我怕的是虚假宣传<img src="https://bbs-static.miyoushe.com/static/2024/05/13/0b9b4077bb63bfd00c6d30e664a16df4_8690624140861603946.png" alt="赛诺-为难" width="75" >

u盘传大文件的这种理想状态才有这个速度，打游戏的4k随机读写就差远了

**Reply by shidijun** [reply]
这样啊<img src="https://bbs-static.miyoushe.com/static/2023/11/16/189d73f83e4892766ac8197f2df156d0_3183903856565806928.png" alt="比利-呜呜呜" width="75" >

**Comment by BlackBee114514** [comment]
其实可以用固态硬盘+硬盘盒的方式 之后扩展也很方便 只是对接口的传输速率有要求 一般顺序300mib/s应该够用（也就是usb3.1/2罢）

**Reply by shidijun** [reply]
好的<img src="https://bbs-static.miyoushe.com/static/2024/09/04/b5afa882060e0ac935cd2ba20a71a334_7137308938668973641.png" alt="卡齐娜-收到" width="75" >

**Comment by ya1355** [comment]
不太建议用U盘来运行游戏或者操作系统，本身U盘散热能力就不行，哪怕只是插在电脑上什么都不做过一会都会很烫，如果经常读取会导致持续高温降低颗粒寿命，如果真需要尽量用机械或者固态移动硬盘

**Reply by shidijun** [reply]
学到了<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by lantainAirblue** [comment]
建议买一个移动硬盘

**Comment by LMA936** [comment]
试过用外接的移动硬盘玩原铁绝，绝的加载时间变长对体验的影响最大，铁次之，反倒是原没有啥感觉，很神奇。

**Reply by LMA936** [reply]
当然，对比的是我笔记本内置的SSD
